examples/tutorials/api/observation_clustering.py

```diff
@@ -0,0 +1,196 @@
+"""
+Observational clustering
+========================
+
+Clustering observations into specific groups.
+
+
+Context
+-------
+
+Typically, observations from gamma-ray telescopes can span a number of
+different observation periods, therefore it is likely that the observation
+conditions and quality are not always the same. This tutorial aims to provide
+a way in which observations can be grouped such that similar observations are grouped
+together, and then the data reduction is performed.
+
+
+Objective
+---------
+
+To cluster similar observations based on various quantities.
+
+Proposed approach
+-----------------
+
+For completeness two different methods for grouping of observations are shown here.
+
+- A simple grouping based on zenith angle from an existing observations table.
+
+- Grouping the observations depending on the IRF quality, by means of hierarchical clustering.
+
+"""
+
+
+import numpy as np
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+import matplotlib.pyplot as plt
+from gammapy.data import DataStore
+from gammapy.data.observations import Observations
+from gammapy.data.utils import get_irfs_features
+from gammapy.utils.cluster import hierarchical_clustering
+
+######################################################################
+# Obtain the observations
+# -----------------------
+#
+# Create the data store and obtain the observations from the H.E.S.S.
+# DL3-DR1 for PKS 2155-304.
+#
+
+data_store = DataStore.from_dir("$GAMMAPY_DATA/hess-dl3-dr1")
+obs_id = data_store.obs_table["OBS_ID"][
+    data_store.obs_table["OBJECT"] == "PKS 2155-304"
+]
```

Comment:
Here I would show some other selections presented in this  [notebook](https://hess-confluence.desy.de/confluence/download/attachments/552632396/analysis_2.ipynb?version=1&modificationDate=1687785248232&api=v2)  including:
- how to access the observations quantities from the 'obs_table' and show a selection from there because it is more convenient than the loop shown after.
- selection based on the "quality" flag

(unrelated: the linked notebook also shows how to remove the observations without valid background files, this could be added to another tutorial/howto if we don't already have that exposed)

Reply:
Okay, I will adapt accordingly

For the quality cut, doesn't this also relate to what you mention about background?

Reply:
the value quality flag  is independent of the background model, @bkhelifi can probably explain how it is defined more in details.